### ISSUE TITLE
Structural difference between World and French version

### ISSUE THREAD

**Issue by nicolasleger** [openfoodfacts/openbeautyfacts#68]
- https://fr.openbeautyfacts.org/marques

<img width="703" alt="screen shot 2018-02-06 at 21 48 16" src="https://user-images.githubusercontent.com/570901/35883435-7e0d0976-0b87-11e8-97e5-600596c7c0e8.png">

- https://world.openbeautyfacts.org/brands

<img width="705" alt="screen shot 2018-02-06 at 21 48 09" src="https://user-images.githubusercontent.com/570901/35883417-7a77161c-0b87-11e8-884b-4445a5386ea6.png">

**Comment by teolemon** [comment]
@stephanegigandet @hangy really weird. Looks like a CSS issue

**Comment by nicolasleger** [comment]
Also between https://world.openbeautyfacts.org/additives and https://fr.openbeautyfacts.org/additifs

**Comment by hangy** [comment]
> @stephanegigandet @hangy really weird. Looks like a CSS issue

Nope. The rendered HTML is different (`<ul>` for French, `<table>` for world). I don't know why that could happen, because all language based subdomains should go to the same backend code. Maybe @stephanegigandet sends the `fr` subdomain to a different backend instance?

**Comment by nicolasleger** [comment]
Indeed https://es.openbeautyfacts.org/marcas looks good
